Source/Urho3D/Graphics/Model.MDL.cpp

```diff
@@ -0,0 +1,264 @@
+// Copyright (c) 2008-2022 the Urho3D project.
+// Copyright (c) 2017-2024 the rbfx project.
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT> or the accompanying LICENSE file.
+
+#include "../Precompiled.h"
+
+#include "../Core/Context.h"
+#include "../Graphics/Geometry.h"
+#include "../Graphics/IndexBuffer.h"
+#include "../Graphics/Model.h"
+#include "../Graphics/Graphics.h"
+#include "../Graphics/VertexBuffer.h"
+#include "../IO/Log.h"
+#include "../IO/File.h"
+#include "../IO/FileSystem.h"
+#include "../IO/VirtualFileSystem.h"
+#include "../Resource/ResourceCache.h"
+#include "../Resource/XMLFile.h"
+
+#include "../DebugNew.h"
+
+namespace Urho3D
+{
+
+bool Model::LoadMDL(Deserializer& source)
```

Comment:
We do not split a class across multiple cpp files. IMO best we do not start doing it :)

Reply:
Why not? If we support several formats for the Model I would prefer the loader code to be isolated. I can't do it as an external loader without major refactoring because of the code for async loading - the loader would need knowledge of and access to internals of the model. I can squash it into one file. Maybe I can extract obj related code that I took from another project to a dedicated cpp...